### ISSUE TITLE
Recommended Mac 3D software (Free/Paid)

### ISSUE THREAD

**Issue by michaeltrilford** [5050frames/core-frame-system#8]
I would like to explore some ideas I have with the C O R E system, but I'm unsure of any good software for MAC.

Any recommendation will be greatly appreciated.

**Comment by 5050frames** [comment]
Hey Michael,

Sorry for the delayed response, just getting caught up. We mostly recommend Fusion 360 for a cross platform solution. That's what I'm using for all the 50/50 parts now.

**Comment by michaeltrilford** [comment]
Fantastic, that’s what I’m using too.

However, the files I imported from this GitHub are not working files? Maybe it is intentional, but I keen to access the ‘sketch’ aspects of the file, so I can ensure the print part is correct.

I’m very new to 3D, but this is what I managed to put together so far. Also printing a test!

![63AA5B1E-EA42-4D9B-AD9E-CA4A54989C3F](https://user-images.githubusercontent.com/2483515/164626687-5a68070e-1eea-4b03-8db8-04815fd967b8.jpeg)
![2DEDB81F-1037-43BD-A781-F9073323AD52](https://user-images.githubusercontent.com/2483515/164626697-0cf7a1f1-7372-4a8f-8122-4d0fbc74c6e3.jpeg)
